tests/conftest.py

```diff
@@ -243,3 +251,50 @@ def _keyfunc(obj):
         return cls.__module__, cls.__name__, obj.name
 
     return sorted(iterable, key=_keyfunc)
+
+
+@pytest.fixture
+def tmp_dir() -> Path:
```

Comment:
You can use [built-in pytest fixtures](https://pytest.org/en/latest/tmpdir.html) for temp directories and files.